charts/flyte-core/templates/redis/service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.redis.enabled }}
```

Comment:
I see we are still installing redis... you will pull this out, right?

Reply:
yes I removed the Redis from flyte-core